packages/table/src/Row.tsx

```diff
@@ -117,6 +118,7 @@ const Row = React.forwardRef(
     }: RowProps,
     ref: React.Ref<any>,
   ) => {
+    const { isBrowser } = useSSR();
```

Comment:
I see this causing problems with SSR testing. The reason is because `useSSR`'s implementation checks `window`, `document`, etc. only once during module load time and then stores the result. This would interact badly with SSR tests because they rely on first mocking `window` etc. to be undefined to simulate server render and then unmocks them to hydrate. Unmocking `window` etc. would have no effect because `useSSR` has already stored the result of the check.

Reply:
Is there a way to update the tests to be more effective in this regard? If this interacts poorly with the tests, then the tests aren't effectively emulating the environments.

Reply:
the most straightforward approach I can think of is to try to mock the `use-ssr` module in tests

to truly simulate the environments and avoid these kinds of problems we would need to have a separate server environment node process that the test process could communicate with in order to avoid the two environments from sharing the same set of loaded modules

Reply:
Wouldn't mocking `use-ssr` possibly defeat the purpose of our SSR tests?

Reply:
if the purpose of SSR tests is to test the behavior of components rendered in an environment without access to window, document, or DOM apis and the subsequent hydration of those components, then mocking `use-ssr` to produce the behavior it should have in each environment would bring us closer to that purpose

Reply:
Perhaps instead of using useSSR we just check if `typeof window !== undefined` like we do elsewhere around the repo, which would mean we don't have to update the jest config and would be more consistent with how we usually handle this same problem

Reply:
I'm probably gonna take a stab at mocking this. I think that useSSR is a slightly more robust solution to this sort of problem, and I'd like to use it more, not less.

Reply:
Turned out we've actually solved this _exact_ issue before with Emotion! The actual fix to make this work was extremely simple, and I've tested that it detects the environment how we expected it to.